app/templates/roadmap/course-ideas.hbs

```diff
@@ -1,13 +1,21 @@
-<AlertWithIcon class="mb-6">
-  Challenges are step-by-step coding exercises where you build projects from scratch. Vote and help us decide which challenges to build.
-</AlertWithIcon>
+<div class="flex flex-col md:flex-row-reverse gap-3">
+  <div class="w-full md:w-60 flex-shrink-0">
+    <div class="md:sticky md:top-4 flex flex-col gap-3">
 
-<div class="grid grid-cols-1 gap-3 mb-8">
-  {{#each this.orderedCourseIdeas as |courseIdea|}}
-    <RoadmapPage::CourseIdeaCard @courseIdea={{courseIdea}} />
-  {{/each}}
+      <LatestReleasesCard @courseExtensionIdeas={{this.releasedCourseExtensionIdeas}} @courseIdeas={{this.releasedCourseIdeas}} />
```

Comment:
```suggestion
      <LatestReleasesCard @courseExtensionIdeas={{this.releasedCourseExtensionIdeas}} @courseIdeas={{@courseIdeas}} />
```

Would just pass in ideas directly, let the component figure out filtering. Makes it harder for us to make mistakes (accidentally pass in unreleased ideas), and reduces the amount of code we need to write overall